### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by jhonny218**
# Objective

Compile all the areas of Speedcurve that we would need to document for creation of a user guide for VSP\VSF teams.

## Areas to Document

- Setup
- Onboarding\Offboarding
- Team setup\Team Admins
- Monthly Budget of Checks
- Default Settings

## Draft

This draft collect all the main setup, configuration and setting for new onboarding teams into SpeedCurve

### CODE REVIEW

File: platform/engineering/frontend/speedcurve/user-guide.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+![Team](../../images/speedCurve-teams.png)
+![Add a new user](../../images/speedCurve-add-a-new-user.png)
+
+### Configure the default settings <a name='default-settings'></a>
+
+Upon the creation of a new team, the site settings need to be adjusted. These are the default settings:
+
+| Setting  |             Default Value              |                                                        Comment                                                        |
+| :------: | :------------------------------------: | :-------------------------------------------------------------------------------------------------------------------: |
+|  Sites   |               [App Site]               |         This should be already included when the team was created.Multiple sites can be added for monitoring          |
```
Comment (billfienberg) [inline platform/engineering/frontend/speedcurve/user-guide.md]: Missing space
```
- This should be already included when the team was created.Multiple sites can be added for monitoring
+ This should be already included when the team was created. Multiple sites can be added for monitoring
```

File: platform/engineering/frontend/speedcurve/user-guide.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+- Analyze the performance impact of ads and third parties.
+
+## Getting started
+
+One user from each veteran-facing services (VFS) team is designated as the administrator for the team. If you are designated as the administrator, you will receive an invitation via email to sign up for SpeedCurve.
+
+Once you are signed in to SpeedCurve, perform the following tasks to configure your team's settings:
+
+1. [Create a team](#create-a-team)
+2. [Add team members](#add-team-member)
```
Comment (billfienberg) [inline platform/engineering/frontend/speedcurve/user-guide.md]: Did you know that Markdown will render the correct list item number in an ordered list for you even if you type the wrong number?

For example:
```
// Markdown
1. [Create a team](#create-a-team)
7. [Add team members](#add-team-member)
4. [Configure default settings](#default-settings)

// Output
1. [Create a team](#create-a-team)
2. [Add team members](#add-team-member)
3. [Configure default settings](#default-settings)
```

Source: https://www.markdownguide.org/basic-syntax/#ordered-lists

I'm not recommending you make that change here. I'm more just letting you know so you can avoid a tedious list update if you have to add more steps to any of these lists in the future.

Comment (jhonnyoddball) [reply platform/engineering/frontend/speedcurve/user-guide.md]: No, I didn't. Thanks for letting me know

File: platform/engineering/frontend/speedcurve/user-guide.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+SpeedCurve also provides performance feedback of each app such as page size and how long it takes to load a page, and can show you the impact new features have on the performance of the app.
+
+SpeedCurve's dashboards provide various perspectives and options. VFS teams can define and monitor performance metrics across VA.gov, test performance in the context of continuous deployment, test responsiveness, and analyze assets. VFS teams can also set a performance budget so teams can see and track when the performance of the site lags.
+
+These are some of SpeedCurve features:
+
+- Track dozens of metrics, including custom metrics.
+- Create performance budgets and get alerts.
+- Diagnose performance issues at the page level (Provided with Google's Lighthouse & PageSpeed Insights audits, there is a list of performance rules, stacked and color-coded in their order of importance.)
+- Analyze the performance impact of ads and third parties.
```
Comment (cvalarida) [inline platform/engineering/frontend/speedcurve/user-guide.md]:
```suggestion
With SpeedCurve, we can:

- Track dozens of metrics
- Create performance budgets and get alerts
- Diagnose performance issues at the page level (Provided with Google's Lighthouse & PageSpeed Insights audits, there is a list of performance rules, stacked and color-coded in their order of importance)
- Analyze the performance impact of ads and third parties
```
I have a thing about not ending bullet points with periods. Don't mind me. :see_no_evil:

File: platform/engineering/frontend/speedcurve/user-guide.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+## Getting started
+
+One user from each veteran-facing services (VFS) team is designated as the administrator for the team. If you are designated as the administrator, you will receive an invitation via email to sign up for SpeedCurve.
```
Comment (cvalarida) [inline platform/engineering/frontend/speedcurve/user-guide.md]:
```suggestion
One user from each Veteran-Facing Services (VFS) team is designated as the administrator for the team. If you are designated as the administrator, you will receive an invitation via email to sign up for SpeedCurve.
```

**Review by cvalarida — APPROVED** [review]
Nicely done! I don't trust my brain today enough to catch anything that may be missing, but from what I can see, it looks great.

**Pull request merged**